api/jira-helpers/projects.js

```diff
@@ -0,0 +1,18 @@
+// const fetch = 1("node-fetch");
```

Comment:
ooops, just realised I checked this file in. It's not operational yet (clearly) but this is kind of the next stage - calling out to JIRA. Happy to delete this file if preferrable.

Reply:
It's not doing any harm.  We can remove it at the next merge or whenever we start to employ the code from it.